cli/commands/tests/ast.test.js

```diff
@@ -1,7 +1,7 @@
 const tmp = require("tmp");
 const path = require("path");
-const { createPackage } = require("../../../cli/commands/new");
-const { printAst } = require("../../../cli/commands/ast");
+const { createPackage } = require("../new");
+const { printAst } = require("../ast");
```

Comment:
Maybe we could export these command functions from an `index.js` inside commands directory. What do you think? It would export just the functions. Not `yargs` stuff.

Reply:
Yeah we can do that. I'll add a commit